mindsdb/integrations/handlers/nuo_jdbc_handler/__init__.py

```diff
@@ -0,0 +1,24 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+try:
+    from .nuo_jdbc_handler import (
+        NuoHandler as Handler,
+        connection_args,
+        connection_args_example,
+    )
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+from .__about__ import __version__ as version, __description__ as description
+
+
+title = 'NuoDB'
+name = 'nuo_jdbc'
```

Comment:
Wouldn't be better if we rename the integration to `nuodb` ?

Reply:
I plan to add another engine for nuodb via the `pynuodb` library. Hence, I vote that we keep the name for this engine as `nuo_jdbc` and the other one as `nuodb`. Let me know if that's okay.